doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -2537,14 +2537,6 @@ \subsubsection{Field: Sky Temperature Schedule Name}\label{field-sky-temperature
 
 This field is used to supply a schedule name of the sky temperature. The sky temperature used in solar radiation calculation of the surface would be overwritten with the value of the scheduled temperature if defined here. If the field is left blank, the global sky temperature would be used.
 
-\subsubsection{Field: Ground View Factor}\label{field-ground-view-factor}
-
-This field defines the constant ground view factor to an external surface. The ground view factor used in solar radiation calculation of the surface would be overwritten with the value of the fraction if defined here. If the field is left blank, the ground view factor of this surface will be calculated with the rules demonstrated above.
-
-\subsubsection{Field: Ground Temperature Schedule Name}\label{field-ground-temperature-schedule-name}
-
-This field is used to supply a schedule name of the ground temperature. The ground temperature used in long wave radiation exchange calculation of the surface would be overwritten with the value of the scheduled temperature if defined here. If the field is left blank, the global ground temperature would be used.
-
```

Comment:
Deleted fields. Replaced with a new object `SurfaceProperty:GroundSurfaces` that allows to specify multiple ground surfaces properties.